### PR TITLE
Adds AmCharts Cursor

### PR DESCRIPTION
**Pull request by mlzummo**
Signed-off-by: Mathew Zummo <mzummo@gmail.com>

### CODE REVIEW

File: src/amchart/CommonXY.js [hunk 1]
```diff
@@ -105,54 +107,60 @@
...
-        this._chart.valueAxes[0].titleFontSize = this.xAxisTitleFontSize();
-        this._chart.valueAxes[0].titleColor = this.xAxisTitleFontColor();
-        this._chart.valueAxes[0].labelRotation = this.xAxisLabelRotation();
-        this._chart.valueAxes[0].autoGridCount = this.xAxisAutoGridCount();
-        this._chart.valueAxes[0].gridPosition = this.xAxisGridPosition();
-        this._chart.valueAxes[0].fillAlpha = this.xAxisFillAlpha();
-        this._chart.valueAxes[0].fillColor = this.xAxisFillColor();
-        this._chart.valueAxes[0].gridAlpha = this.xAxisGridAlpha();
-        this._chart.valueAxes[0].dashLength = this.xAxisDashLength();
+
```
Comment (mlzummo) [inline src/amchart/CommonXY.js]: remove space

File: src/amchart/CommonSerial.js [hunk 2]
```diff
@@ -101,7 +101,7 @@
     CommonSerial.prototype.publish("yAxisType", "linear", "set", "Y-Axis Type", ["none", "linear", "pow", "log", "time"],{tags:["Intermediate","Shared"]});
     CommonSerial.prototype.publish("xAxisType", "ordinal", "set", "X-Axis Type", ["ordinal", "linear", "time"]);
 
-    CommonSerial.prototype.publish("yAxisTickFormat", "s", "string", "Y-Axis Tick Format");
+    CommonSerial.prototype.publish("yAxisTickFormat", ".2f", "string", "Y-Axis Tick Format");
```
Comment (GordonSmith) [inline src/amchart/CommonSerial.js]: Is this the same default as other charts?

File: src/amchart/CommonSerial.js [hunk 3]
```diff
@@ -369,7 +383,19 @@
         var initObj = {
             type: "serial",
             addClassNames: true,
-            chartScrollbar: {}
+            chartScrollbar: {},
+            //precision: 1,
```
Comment (GordonSmith) [inline src/amchart/CommonSerial.js]: remove commented code.

File: src/amchart/CommonSerial.js (hunk 2 above)
Comment (mlzummo) [inline src/amchart/CommonSerial.js]: @GordonSmith 
it's not, i dont think the Axis tick of S is the same as the others either. (however if we dont do any rounding the chart cursor when enabled is to wide and gets clipped for intermediate values) ... (what should we institute as the defaults for the tickformat)?

**Comment by mlzummo** [comment]
changed it to .0f formatting but thats probably not right either ... we need to agree on a default correct formatting ...

**Comment by dtsnell4** [comment]
Looks good

**Pull request merged**